packages/cli/src/utils/auth-files-codegen.js

```diff
@@ -131,17 +161,22 @@ const beforeMiddlewareFunc = (funcName, ...statements) =>
     ),
   );
 
-const afterMiddlewareFunc = (funcName, ...statements) =>
+const afterMiddlewareFunc = (funcName, language, ...statements) =>
   block(
     comment(
       "This function runs after every outbound request. You can use it to check for errors or modify the response. You can have as many as you need. They'll need to each be registered in your index.js file.",
     ),
-    functionDeclaration(funcName, { args: afterMiddlewareArgs }, ...statements),
+    functionDeclaration(
+      funcName,
+      { args: afterMiddlewareArgs(language) },
```

Comment:
Similar to above, `afterMiddlewareArgs` -> `afterMiddlewareArgs(language)` is the actual change here, the rest is formatting.